packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-case-load/main/aspnet/integrationsteps.md

```diff
@@ -31,9 +31,9 @@ The `StartWidgetSigninAsync` call returns a `WidgetSigninResponse` response obje
 ```json
 {
    "interactionHandle":"epXgGYZHsYErPLfw8aLpCvWZOgVtYx25_OYCmQc0z2s",
-   "version":"5.5.2",
+   "version":"${widgetVersion}",
```

Comment:
In this step we don't tell the user what is the widget version until the later steps. Again, not a show stopper for me. Just a comment. Maybe we should move up the "latest version" widget content to this earlier step.

Reply:
Good spot. Will move.